### PR TITLE
Editor/Panel: Fix - Widget search breaks responsivness (ED-2191)

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/scss/editor/panel/_elements.scss [hunk 1]
```diff
@@ -13,9 +13,16 @@
...
+#elementor-global-templates {
 	overflow: hidden;
-	padding: 0 5px;
+	padding: 0 10px;
+	display: grid;
+	gap: 10px;
+	// The uppercase M in `Min` is used in order to avoid SASS conflicts with CSS `min` function.
+	// Ref: https://css-tricks.com/when-sass-and-new-css-features-collide/
+	// Because the gap is 10px, half of the total container width is ( 100% - 10px ) / 2 === ( 50% - 5px ).
+	grid-template-columns: repeat( auto-fill, minmax( Min( 135px, calc( 50% - 5px ) ), 1fr ) );
```
Comment (matipojo) [inline assets/dev/scss/editor/panel/_elements.scss]: Merge with the categories CSS./

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please start the changes from this branch:
https://github.com/elementor/elementor/tree/feature/responsive-panel

**Review by iNewLegend — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

File: assets/dev/scss/editor/panel/_panel.scss [hunk 2]
```diff
@@ -39,10 +39,28 @@
 		width: $panel-xl-width;
 	}
 
-	.ps-scrollbar-y-rail {
+	.ps__rail-y {
 		z-index: 2; // Avoid panel elements (like divider control) from appearing above the scrollbar
 	}
 
+	#elementor-panel-elements,
+	#elementor-global-templates,
```
Comment (kobizz) [inline assets/dev/scss/editor/panel/_panel.scss:47]: `#elementor-global-templates` belongs to the Pro. You should change the rules there

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/_panel.scss (hunk 2 above)
Comment (matipojo) [reply assets/dev/scss/editor/panel/_panel.scss:47]: There is no reason to duplicate the code. it should be managed from one place.

Comment (matipojo) [reply assets/dev/scss/editor/panel/_panel.scss:47]: @StyleShit can you replace it with a general class that can be used on every panel?

Comment (StyleShit) [reply assets/dev/scss/editor/panel/_panel.scss]: Great idea!

File: assets/dev/scss/editor/panel/_panel.scss [hunk 3]
```diff
@@ -39,10 +39,26 @@
...
+		display: grid;
+		gap: 10px;
+		// The uppercase M in `Min` is used in order to avoid SASS conflicts with CSS `min` function.
+		// Ref: https://css-tricks.com/when-sass-and-new-css-features-collide/
+		// Because the gap is 10px, half of the total container width is ( 100% - 10px ) / 2 === ( 50% - 5px ).
+		grid-template-columns: repeat( auto-fill, minmax( Min( 135px, calc( 50% - 5px ) ), 1fr ) );
+	}
+
+	#elementor-panel-elements,
+	#elementor-global-templates {
```
Comment (matipojo) [inline assets/dev/scss/editor/panel/_panel.scss:58]: Add a `TODO` to remove it on 3.2.3.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**